data.sql

```diff
@@ -22,4 +22,42 @@ vet_clinic=# select * from animals;
   4 |  Devimon | 2017-05-12    |               5 | t       |        11
 (4 rows)
 
-*/
\ No newline at end of file
+*/
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (5, 'Charmander', '2020-02-08', 0, false, -11.0, 'Fire Lizard');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (6, 'Plantmon', '2021-11-15', 2, true, -5.7, 'Plant');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (7, 'Squirtle', '1993-04-02', 3, false, -12.13, 'Water Turtle');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (8, 'Angemon', '2005-06-12', 1, true, -45.0, 'Angel');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (9, 'Boarmon', '2005-06-07', 7, true, 20.4, 'Boar');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (10, 'Blossom', '1998-10-13', 3, true, 17.0, 'Flower');
+
+INSERT INTO animals (id, name, date_of_birth, escape_attempts, neutred, weight_kg, species)
+VALUES (11, 'Ditto', '2022-05-14', 4, true, 22.0, 'Transforming');
```

Comment:
As per the project requirements, the insert statements are not supposed to have a value for the species field. This is important as it will affect the transactions and queries being performed. Please update your insert statements to not have values for the species field🙏